scale_build/image/update.py

```diff
@@ -100,7 +100,11 @@ def post_rootfs_setup():
     for binary in filter(lambda s: s == 'apt' or s.startswith('apt-'), os.listdir(binaries_path)):
         binary_path = os.path.join(binaries_path, binary)
         os.chmod(binary_path, stat.S_IMODE(os.lstat(binary_path).st_mode) & no_executable_flag)
-
+    # We want to disable the docker-compose binary so that users want get confused about it
+    # not working correctly on SCALE and push them to use other alternatives like Apps, VM or docker-in-docker
+    ## Please note that this binary is present in two locations and both need to be disabled for correct results.
+    os.chmod(binary_path, stat.S_IMODE(os.lstat("/usr/bin/docker-compose").st_mode) & no_executable_flag)
```

Comment:
What is `binary_path` ? It's been defined above in the loop and will be some apt binary whose perms are being changed here and below